### ISSUE TITLE
vars don't appear to work in commands

### ISSUE THREAD

**Issue by KaelBaldwin** [devspace-cloud/devspace#698]
**What happened?**  
vars are not working in commands

**What did you expect to happen instead?**  
vars should be available for use in commands

**How can we reproduce the bug?** (as minimally and precisely as possible)  
Using the following configuration:
```
commands:
  - name: test
    command: "echo ${USER_NAME}"
vars:
  - name: USER_NAME
    question: What is your user name?
```
run `devspace run test`

Notice you aren't prompted for the USER_NAME variable.
Notice the echo output is empty.

**Local Environment:**  
- Operating System: linux
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: bare-metal
- Kubernetes Version: v1.14.6

**Anything else we need to know?**  

/kind bug

**Comment by FabianKramm** [comment]
@KaelBaldwin Thanks for opening this issue! This is actually intended behavior, vars currently don't work in the vars and commands section, because we don't wanted to interrupt the command execution with a command prompt field.

**Comment by KaelBaldwin** [comment]
@FabianKramm 

I see, as far as the bash interpretation goes, I've seen other projects use an escape mechanism for those situations. Drone for instance uses $${} instead in it's command fields.

I'm having a bit a of trouble now that the namespace field has been removed from the configuration and I was hoping to avoid the users from having to add the --namespace option every time the run devspace dev.

Once I have devspace cloud working fully in our cluster it might handle that part in a space maybe. I haven't figured those out yet.

**Comment by FabianKramm** [comment]
@KaelBaldwin okay I see. The escaping mechanism is a good idea so I think we'll add this to the next release.

**Comment by LukasGentele** [comment]
@KaelBaldwin I think it is valid to add the vars parsing for commands but the namespace should rather be set with `devspace use namespace xyz` which sets the default namespace for the current kube-context which is used by DevSpace during deployment. It's a new command introduced in v4 which should replace the hard coded use of namespace in the config.

**Comment by KaelBaldwin** [comment]
great @LukasGentele thanks, I missed that one.

**Comment by FabianKramm** [comment]
this is implemented in v4.0.2
